controllers/depresolver/depresolver_config.go

```diff
@@ -193,6 +195,10 @@ func (dr *DependencyResolver) validateConfig(config *Config, recognizedDNSTypes
 			return fmt.Errorf("error for geo tag: %s. %s in ns name %s", geoTag, err, nsName)
 		}
 	}
+	err = field(PrometheusAddressPortKey, config.Prometheus.Port).isHigherThanZero().isLessOrEqualTo(65535).isHigherThan(1024).err
```

Comment:
`.isHigherThanZero()` is within `.isHigherThan(1024)` rule

Reply:
@somaritane, good point. Will refactor....